dev/source/docs/mavlink-gimbal-mount.rst

```diff
@@ -27,8 +27,8 @@ Set the mode with MAV_CMD_DO_MOUNT_CONTROL
 The gimbal/mount's mode can be set by sending a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command and param7 fields set as specified for the `MAV_CMD_DO_MOUNT_CONTROL <https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL>`__ command.
 The supported modes are:
 
-0. Retract : gimbal will move to a retracted position (see MNT_RETRACT_X, Y, Z) or relax
-1. Neutral : gimbal will move to a neutral position (see MNT_NEUTRAL_X, Y, Z)
+0. Retract : gimbal will move to a retracted position (see MNT1_RETRACT_X, Y, Z) or relax
```

Comment:
@rmackay9 we need to determine if this legacy command only affects Mount1 or not..the new command specifies the target mount....